runtime/interpreter/errors.go

```diff
@@ -521,3 +521,16 @@ func (e NonStorableStaticTypeError) Error() string {
 		e.Type,
 	)
 }
+
+// InterfaceMissingLocation is reported during interface lookup,
+// if an interface is looked up without a location
+type InterfaceMissingLocation struct {
+	QualifiedIdentifier string
+}
+
+func (e *InterfaceMissingLocation) Error() string {
```

Comment:
Add `Error` suffix like all other errors have:

```suggestion
// InterfaceMissingLocationError is reported during interface lookup,
// if an interface is looked up without a location
type InterfaceMissingLocationError struct {
	QualifiedIdentifier string
}

func (e *InterfaceMissingLocationError) Error() string {
```